clvm/more_ops.py

```diff
@@ -248,7 +257,9 @@ def op_concat(args):
 
 
 def op_ash(args):
-    (i0, l0), (i1, _) = args_as_int_list("ash", args, 2)
+    (i0, l0), (i1, l1) = args_as_int_list("ash", args, 2)
+    if l1 > 4:
+        raise EvalError("ash requires int32 args (with no leading zeros)", args.rest().first())
```

Comment:
I understand you can't use `args_as_int32` on the full arg list, but you might be able to centralize this check by:

>(i0, l0), (i1, l1) = args_as_int_list("ash", args, 2)
> i1 = args_as_int32("ash", args.rest(), 1)

Also redundant, unfortunately - No big deal
:)